Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift

```diff
@@ -106,13 +106,6 @@ final class SentryContinuousProfilerTests: XCTestCase {
         try assertContinuousProfileStoppage()
     }
 
-    func testBackgroundNotificationStopsProfile() {
```

Comment:
`m`: why is this test removed?

Reply:
i moved it and then a merge from upstream brought in the old copy as well. it also exists at https://github.com/getsentry/sentry-cocoa/blob/d21e64c07e78ddf99d12034d0533d8438f9b2f70/Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift#L83